services/venice-controller/src/main/java/com/linkedin/venice/controller/init/ControllerClientBackedSystemSchemaInitializer.java

```diff
@@ -29,15 +35,20 @@
 public class ControllerClientBackedSystemSchemaInitializer {
   private static final Logger LOGGER = LogManager.getLogger(ControllerClientBackedSystemSchemaInitializer.class);
   private static final String DEFAULT_KEY_SCHEMA_STR = "\"int\"";
-  private static final int DEFAULT_RETRY_TIMES = 10;
+  /**
+   * Current leader controller might transit from leader to standby, clear cached store repository, and fail to handle
+   * schema requests. We leverage retries to cover the leader->standby period so that a later retry will be sent to the
+   * new leader controller, who can handle schema requests successfully.
```

Comment:
Maybe it also helps to specify how long the period it can cover with the current setting?

Reply:
Done.